src/parser/xsd/element.js

```diff
@@ -32,7 +32,7 @@ class Element extends XSDElement {
     var descriptor = this.descriptor =
       new XSDElement.ElementDescriptor(qname, typeQName, form, isMany);
 
-    if (this.$nillable) {
+    if (this.$nillable === true || this.$nillable === "true") {
```

Comment:
`this.$nillable` was `false` which is truthy, as a string. Made the conditional more explicit so there are no false positives.

Reply:
can we get this fixed? it solved my issue